### DISCUSSION TITLE
Integration to Home Assistant

### DISCUSSION THREAD

**Discussion by SirNoName2705** [ReVanced/revanced-discussions#1106] [General]
Hey there,
I was thinking about how one could connect ReVanced to a home assistant instance.
Right now, I am thinking of using some kind of docker container that runs android, then installing revanced there, then exposing music/ videos via DLNA.

As you may have guessed by now, I am not quite done with the thinking phase.
Any ideas would be appreciated.

**Comment by oSumAtrIX** [comment]
ReVanced is a patcher for Android apps, not a music or video player. If you're talking about YouTube, the try YouTube asking in related places
